msprime/simulations.py

```diff
@@ -1471,6 +1489,198 @@ def print_history(self, output=sys.stdout):
             self._print_populations(epoch, output)
             print(file=output)
 
+    def population_size_trajectory(self, steps):
+        """
+        This function returns an array of per-population effective population sizes,
+        as defined by the demographic model. These are just the `population_size`
+        parameters of the model, modified by any population growth rates.
+        The sizes are computed at a regular grid of time steps, between time 0 and `end`
+        units of time ago.
+
+        :param int end: Time (ago) at which to stop trajectory inference.
+
+        :param int num_steps: determines at how many time points the
+            coalescence rate is computed.
```

Comment:
delete; add
"
:param list steps: List of times ago at which the population size will be computed.